index.html

```diff
@@ -29,7 +29,127 @@ <h1>I’m Alexandra. Glad to see you!</h1>
+                    <li>CANOPY</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="project-tech-stack">
+                <li><a>HTML</a></li>
+                <li><a>CSS</a></li>
+                <li><a>Javascript</a></li>
+            </ul>
+            <a class="btn">See project</a>
+            </div>
+        </section>
+        <section class="project">
+            <div class="project-header">
+                <img src="./assets/images/project-3.png">
+                <h3 class="project-title">Tonic</h3>
+            </div>
+            <div class="project-details">
+                <ul>
+                    <li>CANOPY</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="project-tech-stack">
+                <li><a>HTML</a></li>
+                <li><a>CSS</a></li>
+                <li><a>Javascript</a></li>
+            </ul>
+            <a class="btn">See project</a>
+            </div>
+        </section>
+        <section class="project">
+            <div class="project-header">
+                <img src="./assets/images/project-4.png">
+                <h3 class="project-title">Multi-Post Stories</h3>
+            </div>
+            <div class="project-details">
+                <ul>
+                    <li>CANOPY</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/images/Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="project-tech-stack">
+                <li><a>HTML</a></li>
+                <li><a>CSS</a></li>
+                <li><a>Javascript</a></li>
+            </ul>
+            <a class="btn">See project</a>
+            </div>
+        </section>
+    </section>
+    
+    <section class="about-myself">
+        <h2 class="title">About Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+            experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a class="contactme">LETS CONNECT</a>
```

Comment:
- [ ] Kindly consider using a heading tag`(hx)` to display the text `LETS CONNECT` as it's a heading instead of the `a` tag. This is for several reasons including convention, semantics and more.